controllers/pipelines/internal/testutil/testutil.go

```diff
@@ -16,3 +19,14 @@ var (
 	TestConfig config.KfpControllerConfigSpec
 	Provider   *pipelinesv1.Provider
 )
+
+func SchemeWithCRDs() *runtime.Scheme {
```

Comment:
```suggestion
func SchemeWithCrds() *runtime.Scheme {
```

not overly precious about it but I prefer strict pascal case even using acronyms